src/routes/posts.js

```diff
@@ -58,6 +58,90 @@ const fetchPosts = async (req, res, next) => {
     }
 }
 
+// ASSIGN TAG TO AN ARTICLE
+
+const assignTagToPost = async (req, res) => {
+    try {
+        const { postId } = req.params // post id;
+        const { tagId, content, title } = req.body
```

Comment:
had a discussion with @jakazzy  and the `content` and `title` being in the `posts_tags` isn't supposed to be so so you'll have to take them out by creating another migration.

A migration is a file that is run to create or update an existing database schema so that it runs for all developers working on the project for uniformity

1. `npm run migrate create drop content and title from posts tag table`. It will create a new migration in the migrations folder
2. use the `exports.up` to define the dropping of the columns. you can use the existing migration as a way to guide you but the specific function we'll be using is here: https://salsita.github.io/node-pg-migrate/#/columns?id=pgmdropcolumns-tablename-columns-options-
3. If everything works well and you start the server, it should successfully remove the columns you specified

Reply:
Okay I will do that

Reply:
So the only columns should be 'postId' and 'tagId' correct?

Reply:
@Yellow-May yeah the only columns should be postId and tagId

Reply:
Okay will implement that